org-alert.el

```diff
@@ -54,7 +57,7 @@
   :group 'org-alert
   :type 'integer)
 
-(defcustom org-alert-notify-after-event-cutoff nil
+(defcustom org-alert-notify-after-event-cutoff 0
```

Comment:
Let's not change any default values, especially if they aren't related to the PR. 0 is a truthy value here and will cause `(if org-alert-notify-after-event-cutoff` to be true but then to disable alerting after the event.

Reply:
yeah, you're right.

I don't quite understand this before and assume that the default behaviour is `0` (especially since I mostly use plain timestamp and didn't really archive my scheduled events).